backend/infrahub/services/adapters/cache/__init__.py

```diff
@@ -34,3 +36,13 @@ async def list_keys(self, filter_pattern: str) -> list[str]:
     async def set(self, key: str, value: str, expires: KVTTL | None = None, not_exists: bool = False) -> bool | None:
         """Set a value in the cache."""
         raise NotImplementedError()
+
+    @classmethod
+    async def new_from_driver(cls, driver: CacheDriver) -> InfrahubCache:
+        module = importlib.import_module(driver.driver_module_path)
```

Comment:
Could we actually use a regular import here instead?
For instance,

```python
if driver == REDIS_DRIVER:
  from redis_driver import RedisDriver
  broker_driver = RedisDriver()
if driver == NATS_DRIVER:
   ...
```

It avoids this tricky way to import and to have hardcoded string paths